kyuubi-server/src/main/scala/org/apache/kyuubi/server/metadata/jdbc/JdbcDatabaseDialect.scala

```diff
@@ -31,6 +32,10 @@ class GenericDatabaseDialect extends JdbcDatabaseDialect {
     s"LIMIT $limit OFFSET $offset"
   }
 
+  override def deleteFromLimitClause(limit: Int): String = {
+    "" // Generic dialect does not support LIMIT in DELETE statements
```

Comment:
let's add a warning log here. it won't bother the user since generally, it won't fall back to `GenericDatabaseDialect`